apps/myaccount/src/components/multi-factor-authentication/authenticators/smsotp-authenticator.tsx

```diff
@@ -62,6 +64,10 @@ export const SMSOTPAuthenticator: React.FunctionComponent<SMSOTPProps> = (props:
         (state: any) => state.authenticationInformation.profileInfo
     );
     const activeForm: string = useSelector((state: AppState) => state.global.activeForm);
+    const [ showMobileUpdateWizard, setShowMobileUpdateWizard ] = useState<boolean>(false);
+
+    // TODO: Get from config
+    const isMobileVerificationEnabled: boolean = true;
```

Comment:
Shall we resolve this TODO ?

Reply:
Let's get this from `deployment.config.json` within the app for time being.

Reply:
Addressed in https://github.com/wso2/identity-apps/pull/1275/commits/375a1a7a8b040739aaae46cb9b9657e461a0bf70